plugins/recruit-resources/src/components/CreateVacancy.svelte

```diff
@@ -143,6 +179,19 @@
       }}
     />
   </svelte:fragment>
+  {#key template?.description}
```

Comment:
We need description in any case, and it by default should be taken from template?.description, but user should be able to create vacancy without template with description.